test/Adapter/Http/ApacheResolverTest.php

```diff
@@ -159,7 +159,29 @@ public function testResolveNoUsers($file)
         $this->assertFalse($result->isValid());
     }
 
-        /**
+    public function testResolveNoValidPasswordWithoutPrependSign()
+    {
+        list($username, $password) = explode(':', base64_decode(base64_encode('admink:admin')));
+        $path = __DIR__ . '/TestAsset/htbasic-append.crypt';
+
+        $this->_apache->setFile($path);
+        $result = $this->_apache->resolve($username, null, $password);
+        $this->assertInstanceOf('Zend\Authentication\Result', $result);
+        $this->assertTrue($result->isValid());
+    }
+
+    public function testResolveNoValidPasswordWithPrependSign()
+    {
+        list($username, $password) = explode(':', base64_decode(base64_encode('admink:admin').'X'));
```

Comment:
This test is pretty much invalid. base64_decode drops `X` here because it does not form full octet.